chapters/data/main.tex

```diff
@@ -409,193 +422,173 @@ \subsection{Measuring variation and importance in our cost components}
 of interpreting this plot is that the further away a large marker is from the
 zero line, the more important that component is to be considered. However, small
 markers are also of interest since these components indicate that the level of
-variation is relatively low \-- the reasons as to why are still unknown.
+variation is relatively low there, perhaps indicating the component has been
+optimised somehow.
 
-It is easily seen from this figure that the conclusions made previously still
-hold. That is, the largest contributors have some of the smallest measures of
-variation while the smallest average contributors are more strongly varied. What
-is of interest is the jump between these groups of cost components. There does
-not seem to be any particular component in the midriff of contributors that has
-large, or indeed small, variation. This suggests that a deeper investigation is
-required to properly analyse individual components and their relationships with
-specific types of patient.
+This figure clearly indicates that the conclusions made previously still hold:
+that the largest contributors have some of the smallest measures of
+variation. Meanwhile, the smallest average contributors are more strongly
+varied. What is of interest is the jump between these components and the others.
+There does not seem to be any particular component in the midriff of
+contributors that has large, or small, variation. As such, a deeper
+investigation is required to properly analyse individual components and
+their relationships with specific types of patient.
 
 
-\section{Taking a slice: diabetic patient analysis}\label{sec:diabetes}
+\section{Diabetic patient analysis}\label{sec:diabetes}
 \graphicspath{{chapters/data/paper/img/diabetes/}}
 
-The main conclusion to be taken away from the previous summative analysis is
-that the dataset contains a huge amount of variation. Therefore, in order to
+The main conclusion to be taken away from the previous analysis was that the
+dataset contains a significant amount of variation. Therefore, in order to
 conduct more meaningful analysis, more homogeneous subsets of the data must be
 considered.
 
-Classically, patients are categorised by age or condition. However, it has been
-shown that doing so often gives an unrepresentative slice of
-patients~\cite{Vuik2016}. In this section, the focus will be on the diabetic
-population within the dataset despite this potential danger as it provides a
-good example of condition-based slicing and is of interest to public health
-groups.
+Classically, patients are categorised by age or condition. However, doing so
+often gives an unrepresentative slice of patients~\cite{Vuik2016a}. In this
```

Comment:
non representative?

(I don't know.)

Reply:
I prefer unrepresentative to non-representative, personally. Would uncharacteristic be better?